contrib/drivers/lvm/lvm.go

```diff
@@ -48,8 +48,9 @@ const (
 )
 
 const (
-	KLvPath  = "lvPath"
-	KLvsPath = "lvsPath"
+	KLvPath     = "lvPath"
+	KLvsPath    = "lvsPath"
+	KLvIdFormat = "NAA"
```

Comment:
Is this fine to keep it all uppercase? Vmware id is in all lowercase naa*

Reply:
Vmware append "naa" for their disk created form a NAA id. keeing "naa" in unique identifer is not required. As per swordfish durableidentifier is an enum , which has "NAA" in caps.
a sample response of swordfish API below. 
"Identifiers": [
{
"DurableNameFormat": "NAA",
"DurableName": "65456765456761001234076100123487"
}
],
We are also targeting the same format.

Reply:
Ok